### ISSUE TITLE
Enhance -g argument to allow multiple/fractional gpus/task

### ISSUE THREAD

**Issue by dylex** [flatironinstitute/disBatch#11]

**Comment by jingpengw** [comment]
Hi @dylex, is this implemented in the beta version.
I think it works in the previous beta version, but it seems not working anymore in the latest beta version for me.

**Comment by dylex** [comment]
I don't think this ever worked, but I also don't think it's relevant anymore in the new implementation that uses slurm tasks, but you'd need to check with Nick.
